myuw/static/vue/components/_common/visual_schedule/schedule-tab.vue

```diff
@@ -180,10 +188,9 @@ export default {
   },
   computed: {
     ...mapState({
-      quarterLastDate: (state) => dayjs(
-          state.termData.lastDay, 'dddd, MMMM D, YYYY',
-      ),
+      quarterLastDate: (state) => dayjs(state.termData.lastDay),
```

Comment:
How do you feel about adding in props:
 term: {
      type: Object,
      required: true,
    },
and replace state.termData to this.term ?